coremltools/test/sklearn_tests/test_feature_names.py

```diff
@@ -25,7 +24,7 @@ def test_all_strings(self):
 
     def test_single_array(self):
         # test both int and long as input to num_dimensions
-        for t in six.integer_types:
+        for t in [int]:
```

Comment:
This is kinda weird but I don't know what the test should be doing.

Reply:
I think we should just get rid of the loop. There is no reason for it any more. We also don't the `t` variable anymore. 

I think the entire body of the function should just be:
```python
self.assertEquals(
    fm.process_or_validate_features("a", num_dimensions=10),
    [("a", dt.Array(10))],
)

```

Reply:
- [ ] Creating a task for this

Reply:
Resolved. Let's hope the CI is happy about this.